specification/maps/data-plane/Search/preview/2022-09-01-preview/examples/GetPolygon.json

```diff
@@ -0,0 +1,627 @@
+{
+  "parameters": {
+    "api-version": "2022-09-01-preview",
+    "location": "47.612560,-122.204141",
+    "resolution": "small",
+    "entityType": "locality"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Content-Type": "application/geo+json"
```

Comment:
>"application/geo+json" [](http://example.com/codeflow?start=24&length=22)

@specialforest will follow up with you offline. We might want to drive consistency between revgc and autosuggest for this.